Makefile

```
@@ -134,7 +134,7 @@ endif
 
 .PHONY: build
 build: generate fmt vet ## Build manager binary.
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build $(GOFLAGS) -mod=mod -o bin/manager main.go
+	CGO_ENABLED=0 GOOS=linux GOARCH=$(shell go env GOARCH) go build $(GOFLAGS) -mod=mod -o bin/manager main.go
```

You may avoid setting GOARCH altogether. You're basically doing GOARCH=GOARCH.

```suggestion
	CGO_ENABLED=0 GOOS=linux go build $(GOFLAGS) -mod=mod -o bin/manager main.go
```